test/integration/ggrc/converters/test_export_csv.py

```diff
@@ -167,6 +169,23 @@ def test_custom_attr_people(self):
     self.assertIn("Included LCAD", response.data)
     self.assertIn("Allowed values are emails", response.data)
 
+  @ddt.data(*(Types.all - Types.external))
```

Comment:
Please use `Types.internal_types` classmethod here.
```suggestion
  @ddt.data(* snapshotter_rules.Types.internal_types())
```